calratio_training_data/constants.py

```diff
@@ -20,3 +20,7 @@
 # Calorimeter Lz is 3500 mm to 6000 mm (abs)
 LLP_Lz_min = 3500
 LLP_Lz_max = 6000
+
+# Min/Max Jet pT 
+min_jet_pt = 40 # GeV
```

Comment:
Please make sure to run `black` and `flake8` on these. I should have that as part of testing!
```suggestion
min_jet_pt = 40  # GeV
```